GeoHealthCheck/config_main.py

```diff
@@ -105,6 +105,7 @@
     'GeoHealthCheck.plugins.probe.http',
     'GeoHealthCheck.plugins.probe.sta',
     'GeoHealthCheck.plugins.probe.wmsdrilldown',
+    'GeoHealthCheck.plugins.probe.wfs3',
```

Comment:
keeping in mind the `wfs3`/`WFS3` acronym may change in the future (TBD), fine for now.

Reply:
yes, first called it `ogcapi.py` but found very few things generic. Changing the name later is a bit tricky as these are also used in configs. But I leave it like this for now, ok?

Reply:
ok